### ISSUE TITLE
NvContainerRestart service is not installed

### ISSUE THREAD

**Issue by spikewj** [loueradun/GamestreamLauncher#7]
I'm using Windows 11 and NVIDIA geforce experience 3.23.0.74

Using GamestreamLauncherConfig.bat, NvContainerRestart service is not installed.
Thus, I tried to install it manually in the cmd with admin priv using the following command but it failed.

D:\GamestreamLauncher-1.3.1\Binaries>InstallUtil.exe NvContainerRestart.exe
Microsoft (R) .NET Framework Installation utility Version 4.7.2556.0
Copyright (C) Microsoft Corporation.  All rights reserved.

Exception occurred while initializing the installation:
System.IO.FileLoadException: Could not load file or assembly 'file:///D:\GamestreamLauncher-1.3.1\Binaries\NvContainerRestart.exe' or one of its dependencies. Operation is not supported. (Exception from HRESULT: 0x80131515).

so that, nvcontainer doesn't restart after gamestream session ends.

In the end, I can install the service with the following command.
SC create "NvContainerRestart" binpath="D:\GamestreamLauncher-1.3.1\Binaries\NvContainerRestart.exe"

then,
sc sdset NvContainerLocalSystem D:(A;;RP;;;AU)(A;;CCLCSWRPWPDTLOCRRC;;;SY)(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;BA)(A;;CCLCSWLOCRRC;;;IU)(A;;CCLCSWLOCRRC;;;SU)(A;;RPWPCR;;;BU)S:(AU;FA;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;WD)

somehow it works but sometimes gamestreamlauncher hangs during running shutting down script.

Is the above commands using "sc create .."  equivalent to the command using InstallUtil.exe?

Is there any way to resolve the installation failure using GamestreamLauncherConfig.bat?

**Comment by loueradun** [comment]
I am not at my desk but that command above looks very similar to what I am
doing now.  It should run an elevated command prompt (which will prompt for
admin access) when saving the config.  Simply opening the config and saving
the settings does this for me.

Another user had mentioned that the service didn't seem to want to install
either, and it may be related to the cmd lines to uninstall any old service
first (I think that fails, so it doesn't continue).

Thanks for the bug report, I will take a look at it when I can.

**Comment by spikewj** [comment]
Thank you for your quick reply!

I read the issue you mentioned, and I see that uninstall && install && sdset .. && sdset will fail when running GamestreamLauncherConfig.bat first time since an exception is thrown during uninstallation.

This issue is regarding the failure of installation command using InstallUtil.exe, which might be due to my environment or windows10, etc.

**Comment by loueradun** [comment]
Yeah I believe I added the && in order to make less calls on the command
line, the uninstall probably shouldn't prefix the statement but should be
its own, as it would continue with the next step if it fails written that
way.  Small change probably, unfortunately I'm underwater with other tasks
atm.
